src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -141,6 +141,38 @@ internal bool IsExceptionType(TypeSymbol type, ref HashSet<DiagnosticInfo> useSi
             return IsEqualOrDerivedFromWellKnownClass(type, WellKnownType.System_Exception, ref useSiteDiagnostics);
         }
 
+        internal bool IsTupleUnderlyingType(TypeSymbol typeToCheck, int requiredArity)
```

Comment:
There are other similar IsXXXX helpers, so I put it in here.
Also since it is likely, that we would need to decouple TupleTypeSymbol from compilation, I did not want to add one more thing that requires compilation. I think location of the method is ok for now, but we may move it appropriately if requirements change.

As for name - I thought that in case we want to relax it to allow not wellknown underlying types it would be more neutral to that. 
I am ok with    IsWellKnownTupleType